.github/workflows/update_fr_template.yaml

```diff
@@ -0,0 +1,48 @@
+name: Update repo from template
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
```suggestion
```

i suggest removing the cron for this initially. can just be an opt-in manual trigger